### PR TITLE
Add gh-pages presubmit jobs for nfd-operator

### PR DESCRIPTION
**Pull request by marquiz**
Add a job for testing the buildability of html documentation and another
one for linting the markdown sources of documentation.

### CODE REVIEW

**Comment by marquiz** [comment]
We should have https://github.com/kubernetes-sigs/node-feature-discovery-operator/pull/50 in place before merging this one
/hold

**Comment by marquiz** [comment]
Rebased, updated k8s test image of build-gh-pages job
/unhold

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Pull request merged**
